wgengine/router/router_linux.go

```diff
@@ -442,95 +552,124 @@ func (r *linuxRouter) delBypassRule() error {
 	if !bytes.Contains(out, []byte(" fwmark "+tailscaleBypassMark)) {
 		return fmt.Errorf("ip rule 10000 doesn't look like a Tailscale policy rule: %q", string(out))
 	}
-	return cmd("ip", "rule", "del", "priority", "10000")
+	return r.cmd.run("ip", "rule", "del", "priority", "10000")
```

Comment:
Unrelated to this particular PR but while I'm reading this code: should we look to see if 10000 is used when the binary starts and warn/error? Exceedingly unlikely, but would be a nightmare to debug.

Reply:
We do look, just above this line: we list rules that have priority 10000, and if any exist, we check if they're shaped like the rule we install. If so, it's a leftover from a previous run and we clean it up, if not we refuse to function because something is interfering with us.